deployment/src/strongmind_deployment/storage.py

```diff
@@ -46,3 +46,30 @@ def __init__(self, name, *args, **kwargs):
                                              acl=acl,
                                              opts=acl_opts
                                              )
+        self.s3_user = aws.iam.User("s3User", name=f"{project}-{stack}-s3User-", tags=tags)
```

Comment:
Instead of a new user, we can attach a policy to the task role for ECS.
See the following docs:
https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/setup-config.html
https://docs.aws.amazon.com/sdkref/latest/guide/feature-container-credentials.html
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html

Reply:
@tpaulshippy I made the changes requested but need some help with the test for the container.